pyeudiw/storage/db_engine.py

```diff
@@ -83,10 +85,58 @@ def update_response_object(self, nonce: str, state: str, response_object: dict)
                     f"Cannot update document with nonce {nonce} and state {state} on {db_name}")
 
         if replica_count == 0:
-            raise Exception(
+            raise ReplicaError(
                 f"Cannot update document with state {state} and nonce {nonce} on any instance")
 
         return replica_count
+    
+    def find_chain(self, entity_id: str) -> Union[dict, None]:
```

Comment:
`find_chain` -> `get_trust_attestation`

Reply:
Resolved in commit [fd3ab55](https://github.com/italia/eudi-wallet-it-python/pull/94/commits/fd3ab555d72149ba44b70c051351a73d4f190dfc)